msteams-platform/apps-in-teams-meetings/build-apps-for-teams-meeting-stage.md

```diff
@@ -194,15 +182,124 @@ The JSON response body for `getAppContentStageSharingCapabilities` API is:
+    const shareSpecificAppContent = (partName) => {
+        var appContentUrl = "";
+        microsoftTeams.app.getContext().then((context) => {
+            appContentUrl = partName == 'todo' ? `${window.location.origin}/todoView?meetingId=${context.meeting.id}` : partName == 'doing' ? `${window.location.origin}/doingView?meetingId=${context.meeting.id}` : `${window.location.origin}/doneView?meetingId=${context.meeting.id}`;
+            microsoftTeams.meeting.shareAppContentToStage((err, result) => {
+                if (result) {
+                    // handle success
+                    console.log(result);
+                }
+
+                if (err) {
+                    // handle error
+                    alert(JSON.stringify(err))
+                }
+            }, appContentUrl);
+        });
+    };
+```
+
+# [View-only screen share](#tab/screen-share-content)
+
+```javascript
+  // Share the content in view-only screen sharing mode.
+      const shareSpecificAppContentScreenShare = (partName) => {
+          var appContentUrl = "";
+          microsoftTeams.app.getContext().then((context) => {
+              appContentUrl = partName == 'todo' ? `${window.location.origin}/todoView?meetingId=${context.meeting.id}` : partName == 'doing' ? `${window.location.origin}/doingView?meetingId=${context.meeting.id}` : `${window.location.origin}/doneView?meetingId=${context.meeting.id}`;
+              microsoftTeams.meeting.shareAppContentToStage((err, result) => {
+                  if (result) {
+                      // handle success
+                      console.log(result);
+                  }
+
+                  if (err) {
+                      // handle error
+                      alert(JSON.stringify(err))
+                  }
+              }, appContentUrl,
+                  // Optional shareOptions with sharingProtocol set to ScreenShare
+                  {
+                      sharingProtocol: microsoftTeams.meeting.SharingProtocol.ScreenShare
+                  });
+          });
+      };
+```
+
+ ---
+
+  | Value | Type | Required | Description |
+  | --- | --- | --- | --- |
+  |`callback` | String | Yes| Callback contains two parameters, error and result. The error can contain either an error of type SdkError or null when share is successful. The result can contain either a true value if there's a successful share or null when the share fails.|
+  |`appContentURL`| String |Yes |The URL that is shared on to the stage.|
+  | `shareOptions`| Object | No | Defines additional sharing options.|
+  |shareOptions.sharingProtocol | Enum | No | The screen sharing protocol. The supported values are `Collaborative` and `ScreenShare`. Default is `Collaborative`.|
+
+  > [!NOTE]
+  > If the value for the `sharingProtocol` property is set as `screenShare`, you don't need to declare any Resource-Specific Consent (RSC) permissions in the app manifest and don't need to set `MeetingStage` in the `getContext` object of the manifest.
```

Comment:
For the API, it will always require the share to stage RSC permission because the developer is programmatically taking over the stage. Suggestion:

```suggestion
  > If the value for the `sharingProtocol` property is set as `screenShare`, you don't need to set `MeetingStage` in the `getContext` object of the manifest. Resource-Specific Consent (RSC) permissions are required in the app manifest for the `shareAppContentToStage` API, regardless of the `sharingProtocol` value used.
```

Reply:
updated